docs/Configuration.md

```diff
@@ -1,21 +1,5 @@
 # Configuration File Format
 
-* [Overview](#overview)
-* [Storage of configurations in JCR](#storage-of-configurations-in-jcr)
-* [Run modes](#run-modes)
-* [Overall structure a of an AC configuration file](#overall-structure-a-of-an-ac-configuration-file)
-* [Configuration of groups](#configuration-of-groups)
-  * [Group migration](#group-migration)
-* [Configuration of users](#configuration-of-users)
-  * [Configuration of keys](#configuration-of-keys)
-    * [Creation of key pair](#creation-of-key-pair)
-    * [Encrypt private key](#encrypt-private-key)
-* [Configuration of ACEs](#configuration-of-aces)
-* [Global Configuration](#global-configuration)
-* [Validation](#validation)
-
-<!--- This table of contents has been generated with https://github.com/ekalinin/github-markdown-toc#gh-md-toc -->
```

Comment:
Did you really want to remove the TOC here?

Reply:
Yes, GitHub renders that automatically now.